src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+function pushMessage(type) {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+
+  if (type === 'error') {
+    message.classList.add('error-message');
+    message.textContent = 'Promise was rejected!';
+  } else {
+    message.textContent = 'Promise was resolved!';
+  }
+
+  document.body.appendChild(message);
+}
+
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
```

Comment:
The `reject` function is called with `Error`, which is a constructor. You should instantiate it by calling `new Error()` to create an error object.